docs/quickstart.rst

```diff
@@ -113,10 +114,12 @@ backed up and that the ``prune`` command is keeping and deleting the correct bac
             /home                           \
             /root                           \
             /var                            \
-                                            \
-        || error "Backup failed" ;
 
-        info "Successfully completed backup
+        backup_exit=$?
+        if [ ${backup_exit} -lt 2 ];
+        then
+            info "Backup successful"
```

Comment:
can we just not give such information?
if you run borg with --show-rc, it will emit a last log line that will tell the exit status "success" or "warning" or "error".

your code would mean borg saying warning and your script saying success and that is confusing.